### PR TITLE
feat : Day 4 - Objects and Array (#39517)

### PR DESCRIPTION
**Pull request by divyansh0777**
@nxanil I have done with task 1 feature on this ticket.  Please have a look!

### CODE REVIEW

File: extra/utils/validation.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  {
+    traineeEmail: 'trainee5@successive.tech',
+    reviewerEmail: 'reviewer5@successsive.tech',
+  }
+]
+
+let pattern = new RegExp ('@successive.tech');
+
+//Validating Email of a user
+let validateEmail = (trainee , reviewer) => {
```
Comment (nxanil) [inline extra/utils/validation.js]: @divyansh0777 At a time only one email will be passed to validateEmail. Please update the function.

File: extra/utils/validation.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    reviewerEmail: 'reviewer4@successive.tech',
+  },
+
+  {
+    traineeEmail: 'trainee5@successive.tech',
+    reviewerEmail: 'reviewer5@successsive.tech',
+  }
+]
+
+let pattern = new RegExp ('@successive.tech');
```
Comment (nxanil) [inline extra/utils/validation.js]: @divyansh0777 Regex is not appropriate, and it just looking for the existence of successive.tech

**Comment by divyansh0777** [comment]
@nxanil  Changes made as you told sir !

**Comment by divyansh0777** [comment]
@nxanil @NishantNaithani1997 done with changes.

**Comment by NishantNaithani1997** [comment]
@divyansh0777 Same question any specific reason for using **let**?

**Comment by divyansh0777** [comment]
@NishantNaithani1997 No there is no specific reason to use **let** but it helps to restrict the scope of variable in block context.

File: extra/utils/permission.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+let permission = {
```
Comment (NishantNaithani1997) [inline extra/utils/permission.js]: @divyansh0777 What if I use _const_?

Comment (divyansh0777) [reply extra/utils/permission.js]:
```suggestion
let permission = {
```
@NishantNaithani1997 we can use **const** as well **instead** of let if we want to.

**Pull request merged**